sdk/compiler/daml-lf-proto-encode/src/DA/Daml/LF/Proto3/EncodeV2.hs

```diff
@@ -721,6 +721,12 @@ encodeUpdate = fmap (P.Update . Just) . \case
         update_FetchInterfaceCid <- encodeExpr fetContractId
         pure $ P.UpdateSumFetchInterface P.Update_FetchInterface{..}
     UGetTime -> pure $ P.UpdateSumGetTime P.Unit
+    ULedgerTimeLT e -> do
+        update_LedgerTimeLt <- encodeExpr e
+        pure $ P.UpdateSumLedgerTimeLt (fromJust update_LedgerTimeLt) -- FIXME:
+    ULedgerTimeLE e -> do
+        update_LedgerTimeLe <- encodeExpr e
+        pure $ P.UpdateSumLedgerTimeLe (fromJust update_LedgerTimeLe) -- FIXME:
```

Comment:
FIXME: could do with a better way of handling the `Just Expr` instance that's returned